backend-repository/product-service/package.json

```diff
@@ -21,13 +21,17 @@
   "dependencies": {
     "@aws-sdk/client-dynamodb": "^3.282.0",
     "@aws-sdk/client-lambda": "^3.282.0",
+    "@aws-sdk/client-sns": "^3.294.0",
+    "@aws-sdk/lib-dynamodb": "^3.294.0",
     "@aws-sdk/util-dynamodb": "^3.282.0",
     "@middy/core": "^3.4.0",
     "@middy/http-json-body-parser": "^3.4.0",
     "aws-sdk": "^2.1325.0",
+    "aws-sdk-client-mock": "^2.1.1",
```

Comment:
the same question

Reply:
I tried writing unit tests ...since I could not finish that I just left it as it is so that I can learn in future